invenio_cern_sync/users/sync.py

```diff
@@ -188,14 +188,18 @@ def _insert_missing(invenio_users, log_uuid, log_name):
                     f"Skipping user with username starting with `_`: {invenio_user}"
                 )
                 continue
-            _id = create_user(invenio_user)
+
+            with db.session.begin_nested():
+                _id = create_user(invenio_user)
+                db.session.flush()
```

Comment:
This should not be needed? the `begin_nested` will commit the transaction `on_exit`
```suggestion

```

Reply:
Right, it should flush before entering and on exit, so we should be safe, thanks 👍